tests/test_sticky_keys.py

```diff
@@ -172,6 +172,50 @@ def test_sticky_key(self):
             [{KC.N0}, {KC.N0, KC.N2}, {KC.N0}, {KC.N0, KC.N3}, {KC.N0}, {}],
         )
 
+        keyboard.test(
+            'stick and roll',
+            [
+                (0, True),
+                (0, False),
+                (2, True),
+                (3, True),
+                (2, False),
+                (3, False),
+            ],
+            [
+                {KC.N0},
+                {KC.N0, KC.N2},
+                {KC.N2},
+                {KC.N2, KC.N3},
+                {KC.N3},
+                {},
+            ],
+        )
+
```

Comment:
That already exists on line 114: stick, multiple interleaved other.

Reply:
Oh. Then that illustrates the `another is pressed` doc change.